_data/simple-icons.json

```diff
@@ -9724,8 +9724,9 @@
         },
         {
             "title": "VMware",
-            "hex": "607078",
-            "source": "https://myvmware.workspaceair.com/"
+            "hex": "53565A",
+            "source": "https://vmware.com/brand",
+            "guidelines": "https://www.vmware.com/content/dam/brand/images/VMware-logo-guidelines.pdf"
```

Comment:
As files (like PDFs) are subject to change we generally prefer to link to a webpage that has a link to the PDF, so could you change this to the page where you found the PDF (and for completeness, if it's not immediately obvious, where on the page the PDF can be found)?

Reply:
I've updated the commit and changed the guidelines for VMware to the public [media site](https://www.vmware.com/company/news/media-resources.html) that links to the VMware Logo Guidelines PDF previously committed as the link.

Ryan